payments/migrations/0008_gatewayoption_historicalgatewayoption_and_more.py

```
@@ -0,0 +1,87 @@
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created_at', models.DateTimeField(auto_now_add=True)),
+                ('updated_at', models.DateTimeField(auto_now=True)),
+                ('gateway', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='payments.gatewayoption')),
+            ],
+            options={
+                'verbose_name': 'Payment Setting',
+                'verbose_name_plural': 'Payment Settings',
+            },
+        ),
+    ]
```

Ruff (0.14.8)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)